producer/httprequest.go

```diff
@@ -78,9 +82,69 @@ func (prod *HTTPRequest) Configure(conf core.PluginConfigReader) error {
 	prod.encoding = conf.GetString("Encoding", "text/plain; charset=utf-8")
 	prod.rawPackets = conf.GetBool("RawData", true)
 
+	//prod.Log.Debug.Printf(
+	//"protocol=%s, host=%s, port=%s, address=%s, encoding=%s, rawPackets=%d",
+	//	prod.protocol, prod.host, prod.port, prod.address, prod.encoding, prod.rawPackets)
+
+	// Health check to ping the backend with an HTTP GET
+	healthcheck.AddEndpointPathArray(
```

Comment:
We'd have to implement an `.AddHealthCheck()` in each of the `core.Simple***` classes separately since the plugins don't have a common ancestor, and with the proposed syntax, add some reflection to append the function name to the endpoint's path, but it could save a couple of lines per endpoint.